pyproject.toml

```
@@ -33,6 +33,7 @@ where = ["src"]
 
 [project.urls]
 Homepage = "https://github.com/disguise-one/python-plugin"
+Documentation = "https://developer.disguise.one/plugins/python-execution-api/"
 Issues = "https://github.com/disguise-one/python-plugin/issues"
```

Maybe we add Disguise itself as sponsor, see well-known alias at: https://packaging.python.org/en/latest/specifications/well-known-project-urls/#well-known-labels
```suggestion
Issues = "https://github.com/disguise-one/python-plugin/issues"
Sponsor = "https://www.disguise.one"
```